lib/annotator-custom-editor.js

```diff
@@ -1,46 +1,49 @@
-// Generated by CoffeeScript 1.9.1
-(function() {
-  var CustomEditor,
-    extend = function(child, parent) { for (var key in parent) { if (hasProp.call(parent, key)) child[key] = parent[key]; } function ctor() { this.constructor = child; } ctor.prototype = parent.prototype; child.prototype = new ctor(); child.__super__ = parent.prototype; return child; },
-    hasProp = {}.hasOwnProperty;
+'use strict';
 
-  Annotator.Plugin.CustomEditor = CustomEditor = (function(superClass) {
-    extend(CustomEditor, superClass);
+var _classCallCheck = function (instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError('Cannot call a class as a function'); } };
 
-    function CustomEditor() {
-      return CustomEditor.__super__.constructor.apply(this, arguments);
+var _createClass = (function () { function defineProperties(target, props) { for (var i = 0; i < props.length; i++) { var descriptor = props[i]; descriptor.enumerable = descriptor.enumerable || false; descriptor.configurable = true; if ('value' in descriptor) descriptor.writable = true; Object.defineProperty(target, descriptor.key, descriptor); } } return function (Constructor, protoProps, staticProps) { if (protoProps) defineProperties(Constructor.prototype, protoProps); if (staticProps) defineProperties(Constructor, staticProps); return Constructor; }; })();
+
+var _inherits = function (subClass, superClass) { if (typeof superClass !== 'function' && superClass !== null) { throw new TypeError('Super expression must either be null or a function, not ' + typeof superClass); } subClass.prototype = Object.create(superClass && superClass.prototype, { constructor: { value: subClass, enumerable: false, writable: true, configurable: true } }); if (superClass) subClass.__proto__ = superClass; };
+
+Annotator.Plugin.CustomEditor = (function (_Annotator$Plugin) {
+  function CustomEditor() {
+    _classCallCheck(this, CustomEditor);
+
+    if (_Annotator$Plugin != null) {
+      _Annotator$Plugin.apply(this, arguments);
+    }
+  }
+
+  _inherits(CustomEditor, _Annotator$Plugin);
+
+  _createClass(CustomEditor, [{
+    key: 'pluginInit',
+    value: function pluginInit() {
+      var _this = this;
```

Comment:
also, for ES6, we don't need to do `var _this = this;`, the fat arrow functionality will take care of this for us – 

can replace the function calls with fat arrow – like instead of 
```
this.annotator.subscribe('beforeAnnotationCreated', function (annotation) {
  _this.customSetup(annotation);
})
```
we can do 
```
this.annotator.subscribe('beforeAnnotationCreated', (annotation) => {
  this.customSetup(annotation);
})
```
which keeps the `this` binding consistent